src/main/java/net/pms/dlna/MediaMonitor.java

```diff
@@ -20,24 +20,22 @@
 import org.slf4j.LoggerFactory;
 
 public class MediaMonitor extends VirtualFolder {
-	private static Set<String> watchedEntries;
+	private static Set<String> fullyPlayedEntries;
 	private File[] dirs;
 	private PmsConfiguration config;
-	private static final FileUtils FILE_UTILS = FileUtils.getInstance();
```

Comment:
I'm not sure if MediaMonitor is always just one thread (even with multiple renderers playing back), but having shared static instances is a pain synchronization wise. I recon move to trash/recycler isn't that frequent an operation and that the avoided synchronization issues is worth the extra construction.